cardano-cli/src/Cardano/CLI/EraBased/Options/Governance/Actions.hs

```diff
@@ -10,7 +10,7 @@ import           Cardano.Api
 import           Cardano.Api.Ledger
 import           Cardano.Api.Shelley
 
-import           Cardano.CLI.EraBased.Commands.Governance.Actions
+import qualified Cardano.CLI.EraBased.Commands.Governance.Actions as Cmd
```

Comment:
The `Cmd` qualification allows us to avoid naming conflicts.  For example in the records we have an `eon` field which if imported unqualified will name conflict with a lot of code.